app/controllers/api/moves_controller.rb

```diff
@@ -19,6 +19,10 @@ def show
 
     def create
       move = Move.new(new_move_attributes)
+
+      move.profile.documents = Document.where(
+        id: (new_move_params.dig(:relationships, :documents, :data) || []).map { |doc| doc[:id] },
+      )
```

Comment:
I don't want to push you to create a service, but we should simplify this,
maybe you can just create a private method for
`new_move_params.dig(:relationships, :documents, :data) || []).map { |doc| doc[:id] }`
or maybe you can move the

```Document.where(
        id: (new_move_params.dig(:relationships, :documents, :data) || []).map { |doc| doc[:id] }
```
into the model?

Reply:
Probably in v2 we can do that if it stays this complex.